packages/flutter/lib/src/services/raw_keyboard.dart

```diff
@@ -113,8 +146,10 @@ class RawKeyEventDataFuchsia extends RawKeyEventData {
 ///
```

Comment:
(not part of your PR, but since you're in here): I believe there is a typo in the sentence above: they -> them?